commands/inventory/add_service_mysql_test.go

```diff
@@ -0,0 +1,79 @@
+// pmm-admin
+// Copyright 2019 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package inventory
+
+import (
+	"testing"
+
+	"github.com/percona/pmm/api/inventorypb/json/client/services"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAddServiceMySQL(t *testing.T) {
+	t.Run("Address and port", func(t *testing.T) {
+		res := &addServiceMySQLResult{
+			Service: &services.AddMySQLServiceOKBodyMysql{
+				ServiceID:      "/service_id/1",
+				ServiceName:    "MySQL Service",
+				NodeID:         "/node_id/1",
+				Address:        "127.0.0.1",
+				Port:           3306,
+				Environment:    "environment",
+				Cluster:        "mysql-cluster",
+				ReplicationSet: "mysql-replication-set",
+				CustomLabels:   map[string]string{"key": "value", "foo": "bar"},
+			},
+		}
+		expected := `MySQL Service added.
```

Comment:
I think we should standardise on:
```go
	expected := strings.TrimSpace(`
multi
line
text
	`) + "\n"
```

That way the whole text block looks natural, and closing `)` is aligned.